README.md

```diff
@@ -47,6 +47,7 @@ helpful to start contributing to the Julia codebase.
 - [**StackOverflow**](https://stackoverflow.com/questions/tagged/julia-lang)
 - [**Twitter**](https://twitter.com/JuliaLanguage)
 - [**Meetup**](https://julia.meetup.com/)
+- [**Learn Julia on JuliaAcademy**](https://juliaacademy.com)
```

Comment:
Maybe this:
```suggestion
- [**Learning resources**](https://julialang.org/learning/)
```
Links to the learning page on JuliaLang.org which lists Julia Academy as the first resource, along with many others.